packages/deployer/test/sample-project/contracts/storage/ProxyStorage.sol

```diff
@@ -0,0 +1,30 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+contract ProxyStorageNamespace {
```

Comment:
Can we name these 'ProxyNamespace'? I think 'Storage' is a bit redundant and verbose.